node_modules/oae-core/activity/activity.html

```diff
@@ -20,7 +20,8 @@
     {macro renderSummary(activity)}
         <div class="activity-summary-container clearfix">
             <div class="pull-left activity-entity-thumbnail">
-                ${renderThumbnail(activity.primaryActor)}
+                {var displayOptions = new Object(); displayOptions.hideLink = true}
+                ${renderThumbnail(activity.primaryActor, displayOptions)}
```

Comment:
Rather than creating the `displayOptions` object in the Trimpath template, can we pass it in through js? That's the pattern we're mostly using for displayOptions now